### DISCUSSION TITLE
Something wrong with my command CTRL+3

### DISCUSSION THREAD

**Discussion by rmaltais** [terjeio/ioSender#384] [Q&A]
Hi,

I don't know if I misconfigured my iosender but the CTRL+3 command just doesn't work. The other commands for the jog steps work just fine, except for the last step on this command.

![Capture d’écran 2024-05-19 191833](https://github.com/terjeio/ioSender/assets/41405404/c00a2cd8-4a7d-42cb-b58d-e224a6cc97ab)

[KeyMap2.zip](https://github.com/terjeio/ioSender/files/15371105/KeyMap2.zip)

thank

**Comment by terjeio** [comment]
I am using a laptop for the time beeing and cannot check the numeric keypad 3 key, however when I remap to ctrl + Q it is working for me. There is a bug though, if the flyout is visible and anything in it is clicked none of the shortcuts are working and I have to click in the Grbl tab to take focus away from the flyout to make it work again. I'll fix that.
